### PR TITLE
Full-text search using materialized view PoC

### PR DESCRIPTION
**Pull request by voneiden**
Ref: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1655

I decided to put the SQL in a separate file as the developer experience is much better compared to when it is inlined in a python script. The downside is that I wasn't able to template the weights, so now there's a regex substitution that applies environment variables. 

The qp's are currently `x_full_text` and the supplementary `x_ongoing`.

Additional material: 
* https://helsinkisolutionoffice.atlassian.net/browse/LINK-1536
* https://helsinkisolutionoffice.atlassian.net/browse/LINK-1389

### CODE REVIEW

File: events/management/commands/refresh_full_text.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            else:
+                self.stdout.write(
+                    f"Applied weight override for field {field} to {weight_override}"
+                )
+            sql = replaced_sql
+        return sql
+
+    def handle(self, *args, **options):
+        if options["drop"]:
+            # Print stdout success info
```
Comment (charn) [inline events/management/commands/refresh_full_text.py:65]: Unnecessary comment?

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit:

File: events/management/commands/refresh_full_text.py (hunk 1 above)
Comment (voneiden) [reply events/management/commands/refresh_full_text.py:65]: copilot steering attempt :face_with_head_bandage:

**Comment by voneiden** [comment]
Some test tweaking was needed due to main changes. Also changed `language` -> `x_full_text_language`

**Pull request merged**